### PR TITLE
Availability check through receptor

### PR DESCRIPTION
**Pull request by slemrmartin**
Following #8 and extending by availability check with receptor directive (now receptor:ping, waiting for satellite-specific)

### CODE REVIEW

File: lib/topological_inventory/satellite/connection.rb [hunk 1]
```diff
@@ -40,10 +61,11 @@ def receptor_controller_base_url
         "#{scheme}://#{host}:#{port}"
       end
 
+      # org_id with any number is required by receptor controller
       def identity_header(account_number)
         {
           "x-rh-identity" => Base64.strict_encode64(
-            {"identity" => {"account_number" => account_number}}.to_json
+            {"identity" => {"account_number" => account_number, "internal" => {"org_id" => '000002'}}}.to_json
```
Comment (agrare) [inline lib/topological_inventory/satellite/connection.rb:68]: This is a little weird, do you know why this is?

Comment (slemrmartin) [reply lib/topological_inventory/satellite/connection.rb:68]: I'm waiting for response

**Review by agrare — APPROVED** [review]

**Comment by agrare** [comment]
sorry @slemrmartin just saw this was still in wip (I look at the labels but looks like that's not set up in this repo)  Was this okay to go in?

**Comment by slemrmartin** [comment]
hm, it seems I should delete and refork my repo. 
It should theoretically work with receptor:ping, but there is probably some issue in receptor controller, so I have to wait with tests

**Pull request merged**